websocket/server/server_test.go

```diff
@@ -0,0 +1,104 @@
+package server
+
+import (
+	"errors"
+	"sync"
+	"sync/atomic"
+	"testing"
+
+	"github.com/multiversx/mx-chain-communication-go/testscommon"
+	"github.com/multiversx/mx-chain-communication-go/websocket"
+	"github.com/multiversx/mx-chain-communication-go/websocket/data"
+	"github.com/multiversx/mx-chain-core-go/core/mock"
```

Comment:
please remove all references to `github.com/multiversx/mx-chain-core-go/core/mock` package and implement them in this repo

Reply:
this will be made in a future PR